composer.lock

```diff
@@ -4887,12 +4887,12 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/symfony/contracts.git",
-                "reference": "b3b9b70bfacfd204d352e7e7350e02c02c0fff78"
+                "reference": "cf1da70384da0d4e98593760955c4397a22175cc"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/symfony/contracts/zipball/b3b9b70bfacfd204d352e7e7350e02c02c0fff78",
-                "reference": "b3b9b70bfacfd204d352e7e7350e02c02c0fff78",
+                "url": "https://api.github.com/repos/symfony/contracts/zipball/cf1da70384da0d4e98593760955c4397a22175cc",
+                "reference": "cf1da70384da0d4e98593760955c4397a22175cc",
                 "shasum": ""
```

Comment:
@guite this is the weird change I was talking about.

Reply:
Seems `b3b9b70bfacfd204d352e7e7350e02c02c0fff78` is a bit outdated:

https://github.com/symfony/contracts/compare/b3b9b70bfacfd204d352e7e7350e02c02c0fff78...cf1da70384da0d4e98593760955c4397a22175cc